python_modules/dagster/dagster/cli/api.py

```diff
@@ -398,7 +399,8 @@ def grpc_command(
     if not (port or socket and not (port and socket)):
         raise click.UsageError("You must pass one and only one of --port/-p or --socket/-s.")
 
-    logger = default_system_logger("dagster-code-server", coerce_valid_log_level(log_level))
+    configure_loggers(log_level=coerce_valid_log_level(log_level))
+    logger = logging.getLogger("dagster.grpc.server")
```

Comment:
I like dagster-code-server (or dagster.code-server or something) better than grpc.server personally - i've been thinking about doing a bigger push to stop calling them 'grpc servers' everywhere. (Similar to how dagit is called 'dagit' not 'the flask server' - grpc is an implementation detail)

Reply:
I think it makes sense to send the log lines to the same logger as `grpc.server`, since the module is currently referenced that way. While I'd be in favor of renaming the module, its probably best to put that out of scope for this PR.

That brings up another point though that specifying the logger names literally might end up leaving easy to miss bugs for if(when) we do actually refactor. We could determine the logger path programmatically with something like `logger = logging.getLogger(inspect.getmodule(DagsterGrpcServer).__name__)`

The effect of this is that if the import path changed, we wouldn't need to change the string as well.